backend/src/routers/donors.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const router = new express.Router();
+const donorController = require('../controller/donors.controllers.js');
+const verifyToken = require('../middleware/verifyAdminToken.js')
+
+router.post('/addDonor',verifyToken, donorController.addDonor);
+router.get('/getDonors', verifyToken, donorController.getDonors);
+router.get('/filterSearch/:searchTerm',verifyToken,  donorController.getFilteredDonors);
+router.delete('/removeDonor/:id',verifyToken, donorController.removeDonor);
+router.patch('/updateDonor',verifyToken, donorController.updateDonor);
```

Comment:
use put instead of patch

Reply:
updated patch to put and i have passed id as param and data to be updated as body